internal/model/modeldecoder/nullable/nullable.go

```diff
@@ -27,6 +27,12 @@ import (
 	jsoniter "github.com/json-iterator/go"
 )
 
+var supportedTSFormats = []string{
+	time.RFC3339,
```

Comment:
```suggestion
	"2006-01-02T15:04:05Z07:00", // time.RFC3339,
```

Might be worth writing out the layout here to highlight the difference to the ones below?